nncf/openvino/optimized_functions/models.py

```diff
@@ -245,6 +245,7 @@ def get_compress_decompress_weight_model(
     zero_point_shape: Optional[Tuple] = None,
     reduction_axes: Optional[ReductionAxes] = None,
     return_compressed_weight: Optional[bool] = False,
+    return_nodes: Optional[bool] = False,
```

Comment:
Do you really need to introduce this argument? Where do you use it?

Reply:
Thanks for the answer! Sorry for being persistent, but I asked because I couldn't find where `get_compress_decompress_weight_model` was used with `return_nodes=True`.

Reply:
Thanks. I see what you mean. Removed.